checklist.md

```diff
@@ -0,0 +1,108 @@
+# Сheck your code against the following points:
+
+1. **CODE EFFICIENCY:** use class name to get access to the class atrributes:
```

Comment:
1. **CODE EFFICIENCY:** use the class name to get access to the class attributes: